app/models/loader/types.rb

```diff
@@ -170,16 +185,33 @@ def identifier
     class Group < Record
       def_delegators :@policy_object, :gidnumber
 
+      def verify; end
+
       def create!
         self.annotations ||= {}
         self.annotations["conjur/gidnumber"] ||= self.gidnumber if self.gidnumber
 
         super
       end
     end
-    
+
     class User < Record
-      def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :restricted_to
+      def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :uidnumberabc, :restricted_to
+
+      # Below is a sample method verifying policy data validity
+      def verify
+        # if self.uidnumber == 8
+        #  message = "User '#{self.id}' has wrong params"
+        #  raise Exceptions::InvalidPolicyObject.new(self.id, message: message)
+        # end
+      end
+
+      # Below is a sample method filling defaults for User entity in policy
+      # def calculate_defaults!
+      #  if self.uidnumber == nil
+      #    self.annotations["conjur/uidnumber"] = 10
+      #  end
+      # end
```

Comment:
I don't think I'm in support of this PR if it's only adding this for future use. Do you have verifications and default values you are planning to add to a type here?

If so, can we include at least one of those with this PR?

Reply:
I agree with you, but for such thing we need feature signoff and it would be problematic to have it in this stage